os/net/bluetooth/bluetooth.c

```diff
@@ -328,8 +312,11 @@ static int bt_init(void)
 		return ret;
 	}
 
-	bt_finalize_init();
+#ifdef CONFIG_BLUETOOTH_NULL
+	write_local_name("TizenRT");
```

Comment:
I don't know what is local name and why it needed but bt_null is lower device driver of bluetooth, upper doesn't know which device is attached directly. (of course we can add some process to get capability of lower device through the ioctl)
So you have to find another way.
But you already call bt_set_name in bt_enable, why write it once again??

Reply:
It is written for testing purpose with bt_null driver.

Reply:
I have removed the write local  name call from bt_init